src/components/modal/examples/Modal.vue

```diff
@@ -5,18 +5,20 @@
     </h2>
 
     <cdr-modal
-      label="label text is title or aria"
       :opened="opened"
       @closed="closed"
       wrapper-class="wrapper-test-class"
       overlay-class="overlay-test-class"
       data-backstop="modal"
       role="dialog"
+      data-ui="hamburger-modal"
+      aria-labelledby="some-heading"
```

Comment:
Changed this example just because the label & title didn't match.  Also, I wanted to have a version like this to screen reader test.  I did notice the title wasn't reading out on focus with VO with this setup, which is kind of surprising.  May use this to do some later work on focus testing.